fontbe/src/glyphs.rs

```diff
@@ -123,24 +123,51 @@ impl Work<Context, Error> for GlyphWork {
         // Hopefully in time https://github.com/harfbuzz/boring-expansion-spec means we can drop this
         let glyph = cubics_to_quadratics(glyph);
 
-        // TODO refine (submodel) var model if glyph locations is a subset of var model locations
```

Comment:
you removed this TODO but it doesn't seem like you implemented the [getSubModel](https://github.com/fonttools/fonttools/blob/3b9a73ff8379ab49d3ce35aaaaf04b3a7d9d1655/Lib/fontTools/varLib/__init__.py#L273) method to support sparse masters

Reply:
oh I see, the VariationModel::deltas method takes a map of master values keyed by location, which in theory can be "sparse" (not contain values for all the locations, but only some), and then only those actually participate in the delta computation, so perhaps you don't actually need to make a sub-model; is that it?

Reply:
I just realised this is even documented in the `deltas` method's docstring:

```
/// Note that it is NOT required to provide a point sequence for every location known
/// to the variation model.
```

By the way, should that method raise an error if any of the locations in `point_seqs` parameter are not actually present among the VariationModel's master locations? It seems that right now they are simply ignored.

Reply:
It should definitely error if you pass a location that isn't defined!

Reply:
Added a check.